.circleci/deploy.sh

```diff
@@ -85,6 +89,10 @@ function _circleci_deploy() {
 
   kubectl annotate deployments/claim-for-crown-court-defence kubernetes.io/change-cause="$(date +%Y-%m-%dT%H:%M:%S%z) - deploying: $docker_image_tag via CircleCI"
   kubectl annotate deployments/claim-for-crown-court-defence-worker kubernetes.io/change-cause="$(date +%Y-%m-%dT%H:%M:%S%z) - deploying: $docker_image_tag via CircleCI"
+
+  # wait for rollout to succeed or fail/timeout
+  kubectl rollout status deployments/claim-for-crown-court-defence
```

Comment:
Not sure when this will timeout?

Reply:
default is 10 minute. to have more control we need to upgrade kubectl 1.12+ client or set `spec.spec.progressDeadlineSeconds`.

Reply:
these are other issues that i will need to address in another PR.